test/Check/Parser/Json.hs

```diff
@@ -78,23 +78,37 @@ prop_comma_unmatched = property $ do
 
 prop_plainChar_matched :: Property
 prop_plainChar_matched = property $ do
-  nonEscapee <- forAll $ (`notElem` K.escapees) `Gen.filter` Gen.alpha
+  nonEscapee <- forAll $ (`notElem` K.escapees) `Gen.filter` Gen.ascii
   parse K.plainChar "" [nonEscapee] === Right nonEscapee
 
 prop_plainChar_unmatched :: Property
 prop_plainChar_unmatched = property $ do
   escapee <- forAll $ Gen.element K.escapees
   parse K.plainChar "" [escapee] ?== isLeft
 
+prop_escapedChar_matched :: Property
+prop_escapedChar_matched = property $ do
+  nonEscapee  <- forAll $ (`elem` K.escapeeChars) `Gen.filter` Gen.ascii
+  text        <- forAll $ pure ['\\', nonEscapee]
+  parse K.escapedChar "" text === Right (read ['\'', '\\', nonEscapee, '\''])
+
+prop_escapedChar_unmatched_0 :: Property
+prop_escapedChar_unmatched_0 = property $ do
+  c <- forAll Gen.ascii
+  parse K.escapedChar "" [c] ?== isLeft
```

Comment:
No actually.  This parse parses sequences such as `\\` and `\n`, so at least two characters are required.  The first character must be `\`